ui/app/components/clients/dashboard.js

```diff
@@ -74,25 +74,23 @@ export default class Dashboard extends Component {
     );
   }
 
-  // top level TOTAL client counts from response for given date range
+  // API client count data by namespace for date range
```

Comment:
Reordered the functions in this component so that `dashboard.js` matches `current.js` when viewing side by side since both have a lot of duplicate functions